src/api/utils/helpers.py

```diff
@@ -87,23 +88,21 @@ def paginate_items(fetched_data):
 
         data_list = []
         for _fetched_item in fetched_data.items:
-
-            # Serialization of RedemptionRequests
-            if fetched_data.items[0].__class__.__name__ == 'RedemptionRequest':
-                data_item = redemp_request_serializer(_fetched_item)
+            if isinstance(_fetched_item, RedemptionRequest):
+                data_item = serialize_redmp(_fetched_item)
+                data_list.append(data_item)
+            else:
+                data_item = _fetched_item.serialize()
                 data_list.append(data_item)
-
-            data_item = _fetched_item.serialize()
-            data_list.append(data_item)
 
         return response_builder(dict(
             status="success",
-            data=dict(data_items=data_list,
-                      count=len(fetched_data.items),
-                      nextUrl=next_url,
-                      previousUrl=previous_url,
-                      currentPage=fetched_data.page),
-            message="Data fetched successfully."
+            data=data_list,
+            count=len(fetched_data.items),
```

Comment:
Sqlalchemy's pagination object has a `total` attribute. Also `pages` if you'd want to know how many pages of the results there are.

Reply:
Can I add that in my next PR?

Reply:
No problem. I'll remind you.